discord/state.py

```diff
@@ -1612,6 +1612,54 @@ def parse_entitlement_delete(self, data: gw.EntitlementDeleteEvent) -> None:
         entitlement = Entitlement(data=data, state=self)
         self.dispatch('entitlement_delete', entitlement)
 
+    def parse_message_poll_vote_add(self, data: gw.PollVoteActionEvent) -> None:
```

Comment:
You should update the answer count, as well as the self_voted property when you recieve these events.